frontend/packages/kubevirt-plugin/src/utils/validations/template/template-validations.ts

```diff
@@ -73,6 +73,27 @@ export class TemplateValidations {
     });
   };
 
+  // Return a valid bus, return the defaultBus if it's valid.
+  getAValidBus = (defaultBus?: DiskBus): DiskBus => {
+    const allowedBuses = this.getAllowedBuses();
+    if (defaultBus && allowedBuses.has(defaultBus)) {
+      return defaultBus;
+    }
+    if (allowedBuses.size >= 1) {
+      return this.getDefaultBus();
+    }
+    return null;
+  };
+
+  getDefaultBus = (): DiskBus => {
```

Comment:
these two methods are confusing and I am not sure which one I should use on the first look.

I am sorry, but can we revert this to the former state and have just one method called `getDefaultBus`?